flash

```diff
@@ -482,6 +482,21 @@ case "${OSTYPE}" in
     ;;
 esac
 
+add_datetime() {
+  # args: $1: full path name of `user-data` file on ssd
+  # get current UTC date/time
+  current_datetime=$(date -u)
+
+  ed "$1" <<EOF
+  /^runcmd:/
```

Comment:
this breaks as soon as I don't use that exact YAML syntax, no? i.e. if I use `runcmd: ["foo"]` then this modification breaks my file, doesn't it?

Reply:
You're right - this regex is too restrictive. I'll change it to `/^runcmd:.*/` at least. Should check if YAML allows for whitespace between `runcmd` and the `:`.